utils/ipfs.ts

```diff
@@ -9,23 +9,28 @@ export const justHash = (ipfsLink?: string): boolean => {
   return /^[a-zA-Z0-9]+$/.test(ipfsLink || '')
 }
 
-const cidRegex = /ipfs\/([a-zA-Z0-9]+)\/?$/
-export const extractCid = (ipfsLink?: string): string => {
+export const fastExtract = (ipfsLink?: string): string => {
   if (!ipfsLink) {
     return ''
   }
 
-  const match = ipfsLink.match(cidRegex)
-
-  return match ? match[1] : ''
+  return ipfsLink.replace('ipfs://ipfs/', '')
 }
 
-export const fastExtract = (ipfsLink?: string): string => {
+const cidRegex = /ipfs\/([a-zA-Z0-9]+)/
+export const extractCid = (ipfsLink?: string): string => {
```

Comment:
maybe suitable to write unit test here :)

Reply:
done in ipfsUtils.spect.ts : could extractCid